superset/utils/core.py

```diff
@@ -1746,18 +1748,6 @@ def get_time_filter_status(
                     "column": ExtraFiltersTimeColumnType.TIME_ORIGIN,
                 }
             )
-
-    if ExtraFiltersTimeColumnType.GRANULARITY in applied_time_extras:
-        if datasource.type == "druid":
-            applied.append({"column": ExtraFiltersTimeColumnType.GRANULARITY})
-        else:
-            rejected.append(
-                {
-                    "reason": ExtraFiltersReasonType.NOT_DRUID_DATASOURCE,
-                    "column": ExtraFiltersTimeColumnType.GRANULARITY,
-                }
-            )
```

Comment:
We should keep this, no?

Reply:
just the else?

Reply:
Right, the `else`.

Reply:
Or maybe not, `ExtraFiltersReasonType.NOT_DRUID_DATASOURCE` suggests that this whole part of the code is Druid only.

Reply:
Thinking about it more, I feel as though these two are pretty druid specific.